app/src/main/java/org/eu/exodus_privacy/exodusprivacy/utils/CommonExtensions.kt

```diff
@@ -28,3 +31,23 @@ fun Chip.setExodusColor(size: Int) {
     this.setTextColor(textColorStateList)
     this.chipBackgroundColor = backgroundColorStateList
 }
+
+fun Chip.setVersionReport(app: ExodusApplication) {
+    val versionReport = when (app.exodusVersionCode) {
+        app.versionCode -> VersionReport.MATCH
+        0L -> VersionReport.UNAVAILABLE
+        else -> VersionReport.MISMATCH
+    }
+    chipIcon = ContextCompat.getDrawable(context, versionReport.iconIdRes)
+    setOnClickListener {
+        Toast.makeText(
+            context,
+            context.getString(
+                versionReport.stringIdRes,
+                app.exodusVersionCode,
```

Comment:
You can remove version app and version report.
This informations is too difficult to understood for users without context.
And show version app on toast with 50 characters :exploding_head:

Reply:
Haha yes. (Done 6b6b0b97c6a27696e7335800b1106e1aed77b138)